src/Native/Runtime/arm64/ExceptionHandling.S

```diff
@@ -216,7 +238,7 @@
 
         // x2 = GetThread()
 #ifdef FEATURE_EMULATED_TLS
-        GETTHREAD_ETLS_2
+        bl RhpGetThreadETLS2
```

Comment:
These exception handling methods are not performance critical at all. It would be fine to always call `RhpGetThread` directly from here, for both `FEATURE_EMULATED_TLS` and `!FEATURE_EMULATED_TLS`, without going through the multiple levels of macros and wrappers.

Note that callee saved registers are saved by ALLOC_THROW_FRAME. You can stash the arguments in one of them around the call.

Reply:
This can be separate change. The multiple levels of macros are not that bad yet.